### DISCUSSION TITLE
[2021-09-16] 수찾기

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#133] [Algorithm]
[수찾기](https://www.acmicpc.net/problem/1920)

**Comment by chaeyeon4u** [comment]
[수찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_1920.java)

**Reply by chaeyeon4u** [reply]
[수찾기 2](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_1920%20(2).java)

**Reply by chaeyeon4u** [reply]
[수찾기3](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_1920%20(3).java)

**Comment by hyk339** [comment]
[수찾기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/baekjoon/BOJ_1920.java)

**Reply by hyk339** [reply]
[수찾기 복습 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/baekjoon/BOJ_1920_repeat.java)

**Comment by pyunji** [comment]
[수찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_1920.java)

**Comment by leechang4** [comment]
[수찾기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ%201920.java)
